webpack.config.js

```diff
@@ -6,9 +6,12 @@ module.exports = {
   },
   devtool: 'source-map',
   module: {
+    preloaders: [
+      { test: /\.js$/, loader: 'jscs-loader', exclude: /\/(spec|node_modules)/ }
```

Comment:
I'd strip the "-loader" bit to keep it more concise and consistent with the rest of the code